frontend/src/utils/orgs.ts

```diff
@@ -7,6 +7,12 @@ export function isOwner(accessCode?: typeof AccessCode[UserRole]): boolean {
   return accessCode === AccessCode.owner;
 }
 
+export function isAdmin(accessCode?: typeof AccessCode[UserRole]): boolean {
+  if (!accessCode) return false;
+
+  return accessCode >= AccessCode.owner;
```

Comment:
yep, this makes sense! i think we want all the owner checks to be this, essentially an admin check.

Reply:
Should only be one more check of `isOwner` remaining, will address in https://github.com/webrecorder/browsertrix-cloud/pull/558